plugins/de.ovgu.featureide.fm.core/src/de/ovgu/featureide/fm/core/io/xml/XmlFeatureModelFormat.java

```diff
@@ -192,13 +192,31 @@ protected void addDescription(Document doc, IFeature feat, Element fnod) {
 	private boolean checkAttributeList(String attributeName, LinkedList<FeatureAttribute> attributeList) {
 		attributeName = attributeName.toLowerCase();
 		for (final FeatureAttribute fa : attributeList) {
-			if (fa.getName().toLowerCase().equals(attributeName)) {
-				return true;
-			}
+			return (fa.getName().toLowerCase().equals(attributeName));
```

Comment:
I was wrong about this, sorry. It now returns true or false immediately, please revert to the original.